leshan-server-core/src/main/java/org/eclipse/leshan/server/observation/ObservationListener.java

```diff
@@ -30,5 +30,13 @@
      * @param observation the observation for which new data are received
      * @param reponse the lwm2m response received
      */
-    void newValue(Observation observation, ObserveResponse response);
+    void onResponse(Observation observation, ObserveResponse response);
```

Comment:
You're right. I think this is a general issue. We clearly need to spend time to better document our API. This is something we should do for the 1.0.0.